.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,366 @@
+          
+          # Preserve OSS-specific files
+          echo "🔒 Preserving OSS-specific configurations..."
+          
+          # Add OSS-specific .github/workflows if needed
+          mkdir -p .github/workflows
+          
+          # Create OSS-specific CI/CD if not exists
+          if [ ! -f ".github/workflows/npm-publish.yml" ]; then
+            cat > .github/workflows/npm-publish.yml << 'EOF'
+          name: 📦 NPM Publish
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)